db/migrate/migration_utils/customizable_utils.rb

```diff
@@ -102,8 +102,9 @@ def delete_invalid_work_package_custom_values
             OR cft.type_id IS NULL
         SQL
       else
+        str_false = postgres? ? "FALSE" : "'f'"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.